storage.conf

```diff
@@ -25,6 +25,11 @@ additionalimagestores = [
 # certain container storage drivers.
 size = ""
 
+# Path to a FUSE helper to use for mounting the file system instead of mounting it
+# directly.
+#fuse_program = "/usr/bin/fuse-overlayfs"
```

Comment:
Should we call this mount_program rather then fuse.  I see no reason to only do this for fuse executables.
```
#mount_program = "/usr/bin/fuse-overlayfs"
```

Reply:
this was already present in containers/storage.  So I've added a new commit to rename the option name everywhere